adijif/clocks/hmc7044.py

```diff
@@ -27,7 +27,7 @@ class hmc7044(hmc7044_bf):
 
     # Defaults
     _d: Union[int, List[int]] = [1, 3, 5, *range(2, 4095, 2)]
-    _n2: Union[int, List[int]] = [*range(8, 65535 + 1)]
+    _n2: Union[int, List[int]] = [*range(8, 4095 + 1)]
```

Comment:
Can we make this unique per solver? Since CPLEX should work over the extended ranges.